test/templates/tooltip/data.json

```diff
@@ -0,0 +1,17 @@
+{
+    "id": "tooltip",
+    "name": "Tooltip",
+    "properties": {
+        "content": ""
+    },
+    "modifier": {
+        "block": [ "left", "right", "top-left", "top-right", "botom-left", "bottom-right"]
```

Comment:
This should be `arrow` not `block` since the modification is occurring on the element and not the block. There are no block modifications AFAIK.

Also, `bottom-left` is spelled incorrectly.